README.md

```diff
@@ -77,42 +68,36 @@ do not feel limited to these areas if you would like to add your take on microse
 - Microservice health checks
 - Database communications security
 - Code quality, such as unit test coverage
-- Style, such as using [eslint][eslint] and specific rules, such as [Air BnB][airbnb-eslint]
-- Application to showcase using the backend api
+- Style, such as using [ESLint][eslint] and specific rules, such as [Airbnb][airbnb-eslint]
+- Application to showcase using the backend API
 
 # Prerequisites
 - A basic understanding of source code control, [git][git-scm] is required.
-- You must make your code available via a [github][github] account.
-- All javascript shall be written using [ES6][ES6] standards
+- You must make your code available via a [GitHub][github] account.
+- All JavaScript shall be written using [ES6][ES6] standards.
 - You should be familiar with creating data APIs.
 
 # Getting Started
 1. Fork this [repository][repository].
-1. Clone the fork to your personal machine
```

Comment:
@josh-g3 Which markdown interpreter do you use? Markdown actually prefers you don't count your numbered bullets. This displayed fine for me in idea and on github.

Reply:
I'm using Typora and I didn't realize that about MD, but it was only showing 1's for me. Maybe they don't have a 100% implementation. Feel free to change that back.

Reply:
FYI, the markdown linter we use catches this. :) The other changes are all good though!